manim/utils/images.py

```diff
@@ -40,16 +40,16 @@ def get_full_vector_image_path(image_file_name: str | PurePath) -> Path:
     )
 
 
-def drag_pixels(frames: list[np.array]) -> list[np.array]:
+def drag_pixels(frames: list[np.ndarray]) -> list[np.ndarray]:
```

Comment:
We have an useful type alias for these cases: `PixelArray`. `frames` can more generally be any `Sequence` (not `Iterable`, though, because `frames` is being indexed).
```suggestion
def drag_pixels(frames: Sequence[PixelArray]) -> list[PixelArray]:
```